modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -150,6 +155,12 @@ public class RocksDbKeyValueStorage implements KeyValueStorage {
     /** Index column family. */
     private volatile ColumnFamily index;
 
+    /** Timestamp to revision mapping column family. */
+    private volatile ColumnFamily tsToRevision;
+
+    /** Revision to timestamp mapping column family. */
+    private volatile ColumnFamily revisionToTs;
```

Comment:
It is not needed now, but it will be needed soon enough (probably in the next change to metastorage). However, I can remove it and it will not affect anything

Reply:
Let's delete it, because it's not needed now.